Integrations/Slack/Slack.py

```diff
@@ -386,27 +395,113 @@ def mirror_investigation():
     if kick_admin:
         CHANNEL_CLIENT.conversations_leave(channel=conversation_id)
     if send_first_message:
-        CLIENT.chat_postMessage(channel=conversation_id, text='This is the mirrored channel for incident {}.'
-                                .format(investigation_id))
+        server_links = demisto.demistoUrls()
+        server_link = server_links.get('server')
+        CLIENT.chat_postMessage(channel=conversation_id,
+                                text='This channel was created to mirror incident {}. \n View it on: {}#/WarRoom/{}'
+                                .format(investigation_id, server_link, investigation_id))
 
     demisto.results('Investigation mirrored successfully, channel: {}'.format(conversation_name))
 
 
 def long_running_loop():
     """
-    Runs in a long running container - checking for newly mirrored investigations.
+    Runs in a long running container - checking for newly mirrored investigations and answered questions.
     """
     while True:
+        error = ''
         try:
             check_for_mirrors()
+            if ENDPOINT_URL:
+                check_for_answers(datetime.utcnow())
+        except requests.exceptions.ConnectionError as e:
+            error = 'Could not connect to the Slack endpoint: {}'.format(str(e))
         except Exception as e:
             error = 'An error occurred: {}'.format(str(e))
             demisto.error(error)
-            demisto.updateModuleHealth(error)
         finally:
+            if error:
+                demisto.updateModuleHealth(error)
             time.sleep(5)
 
 
+def check_for_answers(now):
+    """
+    Checks for answered questions
+    :param now: The current date.
+    """
+
+    integration_context = demisto.getIntegrationContext()
+    questions = integration_context.get('questions', [])
+    users = integration_context.get('users', [])
+    if questions:
+        questions = json.loads(questions)
+    if users:
+        users = json.loads(users)
+    now_string = datetime.strftime(now, '%Y-%m-%d %H:%M:%S')
+
+    for question in questions:
+        if question.get('last_poll_time'):
+            if question.get('expiry'):
+                expiry = datetime.strptime(question['expiry'], '%Y-%m-%d %H:%M:%S')
+                if expiry < now:
+                    answer_question(question.get('default_response'), question, questions, '')
```

Comment:
it better to add default value for `email` in `answer_question` rather than passing empty string